rlclientlib/live_model_impl.cc

```diff
@@ -83,6 +85,59 @@ namespace reinforcement_learning {
       status);
   }
 
+  int live_model_impl::request_decision(const char* context_json, unsigned int flags, decision_response& resp, api_status* status)
+  {
+    resp.clear();
+    //clear previous errors if any
+    api_status::try_clear(status);
+
+    //check arguments
+    RETURN_IF_FAIL(check_null_or_empty(context_json, status));
+
+    // Ensure multi comes before slots, this is a current limitation of the parser.
+    RETURN_IF_FAIL(utility::validate_multi_before_slots(context_json, nullptr, status));
```

Comment:
Why we do not want to provide trace_logger here (and few below calls - get_slot_count, get_event_ids)?

Reply:
Good catch, thanks!